piecrust/Cargo.toml

```diff
@@ -19,6 +19,7 @@ colored = "2"
 libc = { version = "^0.2", default-features = false }
 region = "3"
 tempfile = "3"
+thiserror = "1.0"
```

Comment:
Since the others above only specify major version, perhaps we should do the same with this one.

Reply:
```suggestion
thiserror = "1"
```